specifyweb/frontend/js_src/lib/components/Security/UserCollections.tsx

```diff
@@ -110,10 +110,8 @@ function UserCollectionsUi({
 
 export function UserCollections({
   user,
-  isAdmin,
```

Comment:
The `isAdmin` boolean was here for a reason. 
We should make sure removing this functionality is what is intended!
I'm saying this because we may not be addressing the _real_ issue by scraping off this parameter!

Conceptually, the Specify 6 `SetCollections` button was disabled when the user is an admin because admins should _always_ have access to every Collection. 
A user didn't have to worry about setting Specify 6 collection access for admins because it was implicit. 

If this assumption should be maintained, then maybe there's a separate bug in how we set Specify 6 collection permissions for newly created admins!

Reply:
@specify/ux-testing What are your thoughts on this? Should you (a user with the correct permissions) be able to modify which Specify 6 collections an administrator has access to? 
In terms of Specify 6 collection permissions and access, how do we define an admin?

I understand that flexibility might be nice, but should this rule be kept in place?

Reply:
Specify 6 admins do not immediately have access to all collections. They are given permission to each collection one at a time. It is not like how it is in Specify 7.

It was not implicit even though admins had the ability to add themselves to any collection they chose. Admins had to give themselves (or have another admin grant) access to a collection as you have to have an agent in each discipline for it to work correctly.

It's not a rule, it's a mistake introduced because the development team did not know admins are not given access to all collections automatically in Specify 6 like they are in Specify 7. It's not a huge deal, it's just an oversight that has caused issues with many users since the introduction of the Security & Accounts panel.

Reply:
discussed: give access to set Collection button when 'is admin' + allow to choose 0 collection.

Reply:
Got it. In this case, we should keep this change and allow users with the correct permissions to modify the Specify 6 collection access/permissions of administrators.